test/plugins/validation/2and3/schema-ibm.js

```diff
@@ -159,6 +159,35 @@ describe('validation plugin - semantic - schema-ibm - Swagger 2', () => {
     expect(res.warnings.length).toEqual(0);
   });
 
+  it('should non return an error for a response schema of type file', () => {
+    const config = {
+      schemas: {
+        invalid_type_format_pair: 'error'
+      }
+    };
+
+    const spec = {
+      paths: {
+        '/pets': {
+          get: {
+            responses: {
+              '200': {
+                description: 'legal response',
+                schema: {
+                  type: 'file'
+                }
+              }
+            }
+          }
+        }
+      }
+    };
+
+    const res = validate({ jsSpec: spec }, config);
+    expect(res.errors.length).toEqual(0);
+    expect(res.warnings.length).toEqual(0);
+  });
+
```

Comment:
```suggestion

  it('should return an error for a response schema with non-root type file', () => {
    const config = {
      schemas: {
        invalid_type_format_pair: 'error'
      }
    };

    const spec = {
      paths: {
        '/pets': {
          get: {
            responses: {
              '200': {
                description: 'legal response',
                schema: {
                  properties: {
                    'this_is_bad': {
                      type: 'file',
                      description: "non-root type of file is bad"
                    }
                  }

                }
              }
            }
          }
        }
      }
    };

    const res = validate({ jsSpec: spec }, config);
    expect(res.errors.length).toEqual(1);
    expect(res.errors[0].path).toEqual([
      'paths',
      '/pets',
      'get',
      'responses',
      '200',
      'schema',
      'properties',
      'this_is_bad',
      'type'
    ]);
    expect(res.errors[0].message).toEqual(
      'Property type+format is not well-defined.'
    );

    expect(res.warnings.length).toEqual(0);
  });

```

Reply:
I like that suggestion and I don't imagine it should be too difficult. I will give it a shot